### ISSUE TITLE
Hey Budspencer!

### ISSUE THREAD

**Issue by DragonSlayer87** [budspencer1/GoldMod-Cube2#2]
Hello budspencer,
I have tried your goldmod and it is just great. A person named FragGoD on sauer let me notice it and since that day im loving it.

Anyways there are some changes/issues with your servermod i would like to comment on. The mapfight does not work, when u vote it picks up another winner with 0 votes. Another thing is, could u add like a option to which weapons can pass flag, example in efficctf if all weapons could pass the flag it would be much better than just rifle, cuz its hard to hit someone with rifle when its chaos in effic. That would be awesome!. Like "rugby mode" pass flag weapons chainsaw,rifle,rocket,grenade,chaingun!.
Hope u took interest in my issue buddy, you are awesome
Glückliches neues Jahr
-DragonSlayer

**Comment by budspencer1** [comment]
sure i'd add this :) the issue with mapfight is already fixed, i only need to committ it to github 
thank you for your report. Happy  new year :)

**Comment by DragonSlayer87** [comment]
Oh man ure so nice ;D thx, if you need a soloution to the weapon pass option you can look at qserv he has got the efficctf pass thing where u can pass with any weapons but im sure you will figure it out :smile:  
Thank You so much and i hope u work fast XD, joking
Ure really nice bud
Yours Sincerly Dragonslayer

**Comment by DragonSlayer87** [comment]
When will u be finished with the pass option? Is it hard to make? I think you will make it bud but if u need help look at ds mod qserv collect that might help ya, but just focus on the rugby file..Deathstar said its not hard to make

**Comment by budspencer1** [comment]
Yes, i made it. Without looking at death's code, but thanks anyway :)

**Comment by DragonSlayer87** [comment]
Oh man ty soo much. I cant wait to try it. Have u made tye changes in github?. Friday is fragday haha. Ile tell fg how nice u are :)

**Comment by DragonSlayer87** [comment]
Dont forget it XD. I dont see any option on the server init. Make changes in github bud. 
Thx mate

**Comment by DragonSlayer87** [comment]
It says last edited 18 days ago. U sure u fixed mapfight and enabled the option pass?

**Comment by budspencer1** [comment]
Hello,

before you write more comments to this repository, please visit https://github.com/budspencer1/eXtremeServer . This is my new server-modification (it's same as goldmod, just newer)
and please don't tell me, what i should do... i have better things to do ;)
regards, 
-budspencer1

**Comment by DragonSlayer87** [comment]
Ty and im sorry. Last question before close:Did u add that weapon pass option?

**Comment by DragonSlayer87** [comment]
Never mind i Love ya bro :))))
Closing topic.

**Comment by budspencer1** [comment]
yes i added this

**Comment by DragonSlayer87** [comment]
It works and its stable. Everything works bud ure just great! :)
